### PR TITLE
feat(position): add position lib and less file

### PR DESCRIPTION
**Pull request by davismartin**
Adds Position Lib for Tooltips on Popovers

Still a work in progess still needs some refactoring i.e equations are dependent on arrows so does not work for something without arrows.
### LGTM's
- [ ] Dev LGTM
- [ ] Dev LGTM

### CODE REVIEW

File: source/_less/position.less [hunk 1]
```diff
@@ -0,0 +1,182 @@
+@import 'vars';
+
+@popover-arrow-outer-width: 8px;
+@popover-arrow-width: 7px;
+@popover-background-color: @gray-0;
+@popover-border-color: @gray-300;
+@popover-shadow-color: fade(@gray-900, 16%);
+
+:host {
```
Comment (mcortez1497) [inline source/_less/position.less]: the `#arrow` doesn't need to be nested in `:host`

**Comment by CITguy** [comment]
Please rebase and correct `yarn lint` errors.

File: source/_less/position.less [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    background-color: @gray-0;
+    border-width: @popover-arrow-outer-width;
+    transform: rotate(-45deg);
+    &:after {
+      background-color: @gray-0;
+      border-width: @popover-arrow-width;
+      content: " ";
+    }
+    &,
+    &:after {
```
Comment (cathysiller) [inline source/_less/position.less]: pls alphabetize css properties in this block

**Comment by CITguy** [comment]
@mcortez1497, is this PR still necessary or can we include the CSS in the other PRs?

**Comment by davismartin** [comment]
closing this since we will be using the refactored position lib and added the less file in the tooltips and popovers prs

**Pull request closed without merging**